ui-kit/src/main/res/drawable/ripple_button_large.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="?android:attr/colorControlHighlight">
```

Comment:
maybe we need to use this from colors ? just in case to have ability to override it from theme ?

Reply:
It's a native one. Let's change it in future if we'll want to have different ripple color